Resources/views/macros.html.twig

```diff
@@ -0,0 +1,45 @@
+{% macro toast(message,type,role = 'status',live = 'polite',delay = 10000, autohide) %}
+
+  {% set icon = 'check' %}
+  {% if type == 'success' %}
+    {% if autohide is null or autohide is not defined %}
+      {% set autohide = true %}
+    {% endif %}
+  {% endif %}
+  {% if type == 'info' %}
+    {% set icon = 'info-circle' %}
+    {% if autohide is null or autohide is not defined %}
+      {% set autohide = true %}
+    {% endif %}
+  {% endif %}
+  {% if type == 'warning' %}
+    {% set icon = 'exclamation' %}
+    {% if autohide is null or autohide is not defined %}
+      {% set autohide = false %}
+    {% endif %}
+  {% endif %}
+  {% if type == 'danger' %}
+    {% set icon = 'exclamation-triangle' %}
+    {% if autohide is null or autohide is not defined %}
+      {% set autohide = false %}
+    {% endif %}
+  {% endif %}
+
+  {% if autohide == false %}
+    {% set autohideMessage = 'data-autohide=false' %}
```

Comment:
Why not just use the true of the false value of autohide as value of the data attribute like this : `data-autohide={{ autohide }}`. Then you can remove this whole block